util/request_source.go

```diff
@@ -52,6 +55,12 @@ func (r *RequestSource) SetRequestSourceType(tp string) {
 	r.RequestSourceType = tp
 }
 
+// IsBackground checks whether the request is a background job.
+func (r *RequestSource) IsBackground() bool {
```

Comment:
Question: Why not let the caller(tidb in most case) directly pass a `is_background` field?

Reply:
I want to directly define `background`  by job type. but let the caller define it is also ok to me.